vyper/cli/vyper_json.py

```diff
@@ -223,7 +223,11 @@ def get_input_dict_interfaces(input_dict: Dict) -> Dict:
 
 def get_input_dict_output_formats(input_dict: Dict, contract_sources: ContractCodes) -> Dict:
     output_formats = {}
-    for path, outputs in input_dict['outputSelection'].items():
+    try:
+        output_selection = input_dict['settings']['outputSelection']
+    except KeyError:
+        output_selection = input_dict['outputSelection']
```

Comment:
We're still in Beta, so I am fine with making this a breaking change instead of supporting older behavior.

In the future, would probably add a [`DeprecationWarning`](https://docs.python.org/3/library/exceptions.html#DeprecationWarning) instead.

Reply:
@jacqueswww thoughts?

Reply:
Should be fine to just make the change now.

Reply:
Sure; change w/o warning.